visualizer/src/Controls/ImageControls/LabeledControls/OpacitySlider.js

```diff
@@ -0,0 +1,57 @@
+import { makeStyles } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import Slider from '@material-ui/core/Slider';
+import Tooltip from '@material-ui/core/Tooltip';
+import Typography from '@material-ui/core/Typography';
+import { useSelector } from '@xstate/react';
+import React from 'react';
+import { useLabeled } from '../../../ServiceContext';
+
+const useStyles = makeStyles(theme => ({
+  opacity: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    gap: '20px',
+    paddingTop: theme.spacing(1),
+  },
+}));
+
+function OpacitySlider() {
+  const labeled = useLabeled();
+  const opacity = useSelector(labeled, state => state.context.opacity);
+
+  const handleOpacityChange = (event, newValue) =>
+    labeled.send({ type: 'SET_OPACITY', opacity: Number(newValue) });
+
+  const handleDoubleClick = event =>
+    labeled.send({ type: 'SET_OPACITY', opacity: 0.3 });
```

Comment:
Double-clicking returns opacity to the default value? Why 0.3?

Reply:
0.3 was the value we used up through the 0.1 release, so keeping it the same makes things look more similar. If the user changes the opacity slider to a different value, then cycling raw -> overlay -> labels with the Z hotkey will use that value instead for the overlay.